static/elements/chromedash-schedule.js

```diff
@@ -45,7 +45,7 @@ class ChromedashSchedule extends LitElement {
   static get properties() {
     return {
       channels: {attribute: false}, // Assigned in schedule.js, value from Django
-      hideBlink: {attribute: false}, // Edited in schedule.js
+      showBlink: {attribute: false}, // Edited in schedule.js
```

Comment:
Is the comment inaccurate there, or do we need to update shedule.js as well?

Reply:
That comment must be out of date because there is no schedule.js.  The file is named schedule-page.js and that is included in this PR.  I updated the comment.